float64 way1_lati
float64 way1_longi
float32 way1_alti

float64 way2_lati
float64 way2_longi
float32 way2_alti

float64 way3_lati
float64 way3_longi
float32 way3_alti